app/assets/stylesheets/partials/_news.scss

```diff
@@ -33,4 +34,8 @@
     margin-top: 5px;
     transform: rotate(37deg);
   }
+
+  h3 {
+    word-break: break-word;
+  }
```

Comment:
You would get the same behaviour if you put `word-break: break-word` directly into the `.news` bracket on top of this page instead of adding it to `p` and `h3` separately.